pkg/stack/unwind/executable.go

```diff
@@ -16,13 +16,73 @@ package unwind
 
 import (
 	"debug/elf"
+	"errors"
 	"fmt"
+	"os"
 	"strings"
+	"syscall"
 
+	burrow "github.com/goburrow/cache"
 	"github.com/hashicorp/go-version"
 	"github.com/xyproto/ainur"
 )
 
+type FramePointerCache struct {
+	cache burrow.Cache
+}
+
+// the inode value can be recycled (this is implementation specific)
+// and it's only unique across filesystems. By adding the creation time
+// to the cache key we significantly reduce the chances of producing wrong
+// results.
+type framePointerCacheKey struct {
+	inode        uint64
```

Comment:
Didn't add the path as it's more storage than we need, but also because I don't think it gives us any advantages. Files can be renamed, symlinks should be handled, etc